crates/bevy_sprite/Cargo.toml

```
@@ -31,3 +31,4 @@ guillotiere = "0.6.0"
 thiserror = "1.0"
 rectangle-pack = "0.4"
 serde = { version = "1", features = ["derive"] }
+wgpu = { version = "0.12.0" }
```

```suggestion
wgpu = "0.12.0"
```

also nit: missing trialing newline

---

fixed